### DISCUSSION TITLE
Error Handling with i18n router app/[lang]/

### DISCUSSION THREAD

**Discussion by naseef0** [vercel/next.js#50195] [Help]
How will it be when it comes to error handling with the i18n router in the /app/[lang]?

**Comment by izoukhai** [comment]
Similar issue have been raised, but still no answers...

ref. https://github.com/vercel/next.js/issues/49686 https://github.com/vercel/vercel/issues/9827

**Comment by indrajithc** [comment]
I'm facing the same issue....
